### PR TITLE
Lesson 8

### PR DESCRIPTION
**Pull request by lolitamalinovskaya**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -178,4 +178,383 @@ const data = [
...
+        return currentYear - this.admissionYear;
+    }
+}
+
+class Students {
+    constructor(students){
+        this.students = students;
+    }
+}
+Students.prototype.getInfo = function (){
```
Comment (dori17) [inline main.js:526]: it's not necessary to write Students.prototype here, you can just write getInfo() {...}

Comment (dori17) [reply main.js:526]: this will be a method in a class with will act differently for each instance depending on 'this'

Comment (dori17) [reply main.js:526]: everything else in this hometask is great! Amazing job!

**Review by dori17 — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (lolitamalinovskaya) [reply main.js:526]: Using Students.getInfo = function() { … } we’re defining a static function, but task says getInfo() should be instance’s function. getInfo() could be defined in a class, but according task text getInfo should be defined as «prototype, function». Maybe I’ve missunderstood the task

**Pull request merged**